src/reducers-actions/pendingOrders.ts

```diff
@@ -0,0 +1,79 @@
+import { PendingTxObj } from 'api/exchange/ExchangeApi'
+import { GP_ORDER_TX_HASHES } from 'const'
+
+export const enum ActionTypes {
+  SAVE_PENDING_ORDERS = 'SAVE_PENDING_ORDERS',
+  REMOVE_PENDING_ORDERS = 'REMOVE_PENDING_ORDERS',
+}
+
+interface Actions<T> {
+  type: ActionTypes
+  payload: T
+}
```

Comment:
Seems very similar to tokenRow actions.

It it worth making a generic Actions interface?

Reply:
:+1: